internal/discover/mounts_test.go

```diff
@@ -188,3 +188,131 @@ func TestMounts(t *testing.T) {
 		})
 	}
 }
+
+func TestMountsDeterministicOrder(t *testing.T) {
```

Comment:
Did you verify that these tests were failing before the changes?

Reply:
No, I added them after the changes, actually.

Reply:
let me reverse the changes on mounts.go file and run the tests.... 1 sec

Reply:
checked, the tests should catch a regression

Reply:
I don't think that a separate test case is required here. We can add a new entry to the table test in `TestMounts` that returns more candidates and replace:
```
require.ElementsMatch(t, tc.expectedMounts, mounts)
```
with
```
require.EqualValues(t, tc.expectedMounts, mounts)
```
since `ElementsMatch` ignores ordering.

Reply:
thanks for the suggestion, added!